database/src/pool.rs

```diff
@@ -210,6 +210,26 @@ pub trait Connection: Send + Sync {
         sha: &str,
         parent_sha: &str,
     ) -> anyhow::Result<()>;
+
+    /// Try and mark the benchmark_request as completed. Will return `true` if
+    /// it has been marked as completed else `false` meaning there was no change
+    async fn try_mark_benchmark_request_as_completed(
```

Comment:
```suggestion
    async fn mark_benchmark_request_as_completed(
```
I think that the word "try" is too overloaded in this context.